### DISCUSSION TITLE
PNP RefinableSearch can't find options

### DISCUSSION THREAD

**Discussion by sayuri8** [microsoft-search/pnp-modern-search#2765] [Q&A]
Hi all,

I searched for RefinableString100, for example, and tried to insert the corresponding column here, but as you can see, the list is empty. What am I doing wrong? Is there a setting that I need to make?

I also tried to add the column in another way, but it doesn't appear in the list of search results.
![553C9B16-425B-42D1-B91E-FD7B5FFE504E](https://user-images.githubusercontent.com/124573411/218427904-e666c6c9-271d-4d5b-94b6-88a23ce0fd0e.jpeg)

**Comment by wobba** [comment]
Take a look at e.g. https://learn.microsoft.com/en-us/microsoft-365/community/how-do-site-columns-become-managed-properties-thus-available-for-search
